src/test/scala/chiselTests/SIntOps.scala

```diff
@@ -85,11 +85,11 @@ class SIntOpsTester(c: SIntOps) extends Tester(c) {
  */
 
 class SIntLitExtractTester extends BasicTester {
-  assert(-5.S(1) === true.B)
-  assert(-5.S(2) === false.B)
-  assert(-5.S(100) === true.B)
-  assert(-5.S(3, 0) === "b1011".U)
-  assert(-5.S(9, 0) === "b1111111011".U)
+  assert(-5.S.apply(1) === true.B)
+  assert(-5.S.apply(2) === false.B)
+  assert(-5.S.apply(100) === true.B)
+  assert(-5.S.apply(3, 0) === "b1011".U)
```

Comment:
this is currently not consistent with the implementation -- you did not deprecate this version so you should not need the `apply` for the 2-argument version. Which way do you want to go?

Reply:
or maybe we should have tests for both versions